plugins/amdgpu/Makefile

```diff
@@ -0,0 +1,14 @@
+all: dummy_plugin.so
+
+dummy_plugin.so: dummy_plugin.c
+	gcc -g -Werror -D _GNU_SOURCE -Wall -shared -nostartfiles dummy_plugin.c -o dummy_plugin.so -iquote ../../../criu/include -iquote ../../criu/include -fPIC
+
+clean:
+	rm dummy_plugin.so
+
+install:
+	mkdir -p /var/lib/criu && cp dummy_plugin.so /var/lib/criu/
```

Comment:
I am not happy with the hardcoded location. I know CRIU defaults to `/var/lib/criu`, but that sounds wrong anyway. Especially when looking at it from a Linux distribution package. Not sure what to here as the plugin infrastructure does not really exist. Also install must respect `$DESTDIR` for Linux distribution package generation.

Reply:
I changed to $PLUGINDIR from criu env https://github.com/RadeonOpenCompute/criu/blob/amdgpu_rfc-210715-2/plugins/amdgpu/Makefile . Does that sound right?